### PR TITLE
Added Api contract test case for tenants endpoint

### PR DESCRIPTION
**Pull request by gokulakrishnansvm**
Adding API Contract Test Case for tenants endpoint.
<hr/>

## Which Traffic Control components are affected by this PR?

- Traffic Ops Test

## What is the best way to verify this PR?

Step 1 : Follow steps from README file(traffic_ops/testing/api_contract) to run api-contract-tests.

Step 2 : To execute tenants API Contract Testcase , navigate to traffic_ops/testing/api_contract/v4
pytest --to-user Username --to-password Password --to-url URL --config Config Path --request-template Request template --response-template Response template test_tenants.py

Note: Config, Request template and response template paths are optional. 

## PR submission checklist
- [x] This PR has tests 
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://apache.org/security) for details)

### CODE REVIEW

File: traffic_ops/testing/api_contract/v4/request_template.json [hunk 1]
```diff
@@ -43,5 +43,12 @@
...
+	],
+	"tenants": [
+		{
+			"active": true,
+			"name": "test",
+			"parentId": 1
+		}
 	]
-}
+}
```
Comment (TerekhovaKate) [inline traffic_ops/testing/api_contract/v4/request_template.json:54]: not newline at the end of file

File: traffic_ops/testing/api_contract/v4/response_template.json [hunk 2]
```diff
@@ -149,5 +149,28 @@
...
+            "lastUpdated": {
+                "type": "str"
+            },
+            "parentName": {
+                "type": "str"
+            }
+        }
     }
-}
+}
```
Comment (TerekhovaKate) [inline traffic_ops/testing/api_contract/v4/response_template.json:176]: no newline at the and of file

File: traffic_ops/testing/api_contract/v4/request_template.json (hunk 1 above)
Comment (gokulakrishnansvm) [reply traffic_ops/testing/api_contract/v4/request_template.json]: Added!

File: traffic_ops/testing/api_contract/v4/response_template.json (hunk 2 above)
Comment (gokulakrishnansvm) [reply traffic_ops/testing/api_contract/v4/response_template.json]: Added!

**Review by TerekhovaKate — APPROVED** [review]

**Pull request merged**